### PR TITLE
DLS-9277 - preventing duplicate returns

### PR DESCRIPTION
**Pull request by TomRafferty**

### CODE REVIEW

**Review by johno1985 — COMMENTED** [review]
you'll need to modify the unit tests and fix the failing PR build

**Comment by TomRafferty** [comment]
rebuild

**Comment by johno1985** [comment]
PR build failed

File: app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/StartController.scala [hunk 1]
```diff
@@ -232,6 +233,9 @@ class StartController @Inject() (
 
       case NewEnrolmentCreatedForMissingEnrolment(subscribedDetails, ggCredId) =>
         handleSubscribedUser(subscribedDetails.cgtReference, ggCredId, Some(subscribedDetails))
+
+      case _ =>
```
Comment (johno1985) [inline app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/StartController.scala:237]: I would remove this for now, just keep the extra case above

**Review by johno1985 — CHANGES_REQUESTED** [review]
comment added

**Comment by TomRafferty** [comment]
rebuild

**Review by johno1985 — APPROVED** [review]

**Pull request merged**